sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/CommonTestFramework/TelemetryItemValidationHelper.cs

```diff
@@ -143,14 +143,6 @@ internal static class TelemetryItemValidationHelper
 
             var expectedTagsCount = 5;
 
-            if (activityKind == ActivityKind.Server)
```

Comment:
This needs to be validated only for HTTP scenario, but this is a manual trace without HTTP semantics hence removed this check.

Reply:
I think operation name should be there for consumer spans too. Will check on it.

Reply:
The logic for extracting the operation name is currently based on HTTP semantic attributes, which may not be suitable for custom spans. You're right, it's important to evaluate whether this logic is necessary for custom spans and determine the appropriate value for them.